daemon/pkg/plugin/backend_linux.go

```diff
@@ -42,6 +41,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// MediaTypePluginConfig specifies the mediaType for plugin configuration.
+//
+// It is a copy of the mediaType defined in [schema2.MediaTypePluginConfig].
+//
+// [schema2.MediaTypePluginConfig]: https://pkg.go.dev/github.com/distribution/distribution/v3@v3.0.0/manifest/schema2#MediaTypePluginConfig
+const MediaTypePluginConfig = "application/vnd.docker.plugin.v1+json"
```

Comment:
@dmcgowan do you think we should add the missing  mediaTypes in the list of types in containerd? At least looks like some of the legacy types are not in that list. I'm slightly on the fence because they're only used basically for detecting old media-types. OTOH, it's nice to have a somewhat central place to have these.


https://github.com/containerd/containerd/blob/87742bd35f6ddc47c638a448c271b7ccf8df9010/core/images/mediatypes.go